packages/templates/wasm/rust/Cargo.toml

```diff
@@ -1,17 +1,22 @@
 [package]
-name = "template-wasm-rs"
+name = "template_wasm_rs"
 version = "0.1.0"
 description = "A Polywrap Wasm wrapper template written in Rust"
 authors = ["Polywrap"]
 repository = "https://github.com/polywrap/monorepo"
 license = "MIT"
 edition = "2021"
 
+include = ["src"]
+
 [dependencies]
-polywrap-wasm-rs = { version = "~0.11.0-pre.4" }
+polywrap-wasm-rs = { version = "~0.11.2" }
 polywrap_msgpack_serde = "~0.0.2-beta.5"
 serde = { version = "1.0", features = ["derive"] }
 
+[dev-dependencies]
+polywrap = { version = "~0.1.8" }
```

Comment:
```suggestion
polywrap = { version = "0.1.9-beta.2" }
```

Reply:
I'm not sure we should use a beta release in the template

Reply:
updated